### ISSUE TITLE
403 - Forbidden | Accessing Asset Models or Suppliers

### ISSUE THREAD

**Issue by H-Dennis** [snipe/snipe-it#5911]
#### Please confirm you have done the following before posting your bug report:

- [ Yes ] I have enabled debug mode 
- [ Yes ] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

**Describe the bug**
After upgrading from v3.6.4 (manual install) to v4.5.0 (Git install), I was running in some issues launching the app. Most of them I was able to solve using the available documentation, but I'm still having trouble opening the "Asset Models" and "Suppliers" page. I receive "403 - Forbidden: Access is denied." on clients and "HTTP Error 403.14 - Forbidden" on the server.
Weird thing is, when I rename the "%websiteroot%\public\models" and "%websiteroot%\public\suppliers" directories, the pages open just fine. I do receive a "Too many redirects" the first time I reload the page.
In the meantime I've updated through Git to the latest version, but I'm still having the issue.

IIS seems to be able to load "/models" just fine, but not "/models/". (see IIS log snippet below)

**To Reproduce**
1. Go to the main page
2. Hover over the sidebar settings-icon
3. Click on Asset Models or Suppliers

**Expected behavior**
Ability to edit asset models and suppliers.

**Screenshots**
Client
![image](https://user-images.githubusercontent.com/41477499/43003902-501e11ea-8c2e-11e8-8ac3-5ca121ceb687.png)

Server
![image](https://user-images.githubusercontent.com/41477499/43003916-628d8c52-8c2e-11e8-9801-8455d4a5b5ce.png)

**Server:**
 - Snipe-IT Version v4.6.2 - build 3844 (master)
 - OS: Windows Server 2012
 - Web Server: IIS (8.0.9200.16384)
 - PHP Version 5.6.22

**Desktop:**
 - OS: Windows 10 Version 1803 (Build: 17134.137)
 - Browser Chrome, Edge, IE
 - Version 67.0.3396.99, 42.17134.1.0, 11.112.17134.0

**Error Messages**
- WITH DEBUG TURNED ON, if you're getting an error in your browser, include that error
> No application error, only web server error 403. (see screenshots)

- If a stacktrace is provided in the error, include that too.
> N/A

- Any errors that appear in your browser's error console.
> Chrome: Failed to load resource: the server responded with a status of 403 (Forbidden)

- Confirm whether the error is reproducible on the demo: https://snipeitapp.com/demo.
> Not reproducible.

- Include any additional information you can find in `storage/logs` and your webserver's logs.
> No info in storage/logs
> 2018-07-20 13:28:13 172.16.31.22 GET /models - 80 - INTERNALIP Mozilla/5.0+(Windows+NT+10.0;+WOW64;+Trident/7.0;+rv:11.0)+like+Gecko URL 301 0 0 0
2018-07-20 13:28:13 172.16.31.22 GET /models/ - 80 - INTERNALIP Mozilla/5.0+(Windows+NT+10.0;+WOW64;+Trident/7.0;+rv:11.0)+like+Gecko URL 403 14 0 9

**Additional context**
- Is this a fresh install or an upgrade? 
> Upgrade

- What OS and web server you're running Snipe-IT on 
> See Server info

- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc) 
> See issue description

- Include what you've done so far in the installation, and if you got any error messages along the way. 
> v3.6.4 was the initial install (manual). We've then imported the assets from an csv. Afterwards not that much has changed. (I was away for over a year.) Now we've updated to v4.5.0 and v4.6.2 through Git.

- Indicate whether or not you've manually edited any data directly in the database
> Not sure...

Please do not post an issue without answering the related questions above. If you have opened a different issue and already answered these questions, answer them again, once for every ticket. It will be next to impossible for us to help you.

**Comment by snipe** [comment]
Hm. That's a server-level error, not an app error. I'd guess something to do with your rewrite settings with regard to trailing slashes? I don't use IIS. :-/

**Comment by H-Dennis** [comment]
Indeed.
I'm also no IIS expert. As far as I remember, I didn't do anything on IIS except for the initial configuration (importing the site), modifying the path and setting the URL to listen to.
I've never even opened the rewrite rules up until now.
I'll see what I can find. :-)

**Comment by H-Dennis** [comment]
Still no progess.
Tried to delete the rewrite rules from IIS, import them again from the .htaccess file and restart the website, but I still have the same issue.

![image](https://user-images.githubusercontent.com/41477499/44027537-2a41df7a-9ef7-11e8-984b-889d02705f41.png)

I needed to input some new asset models, so to get this to work and load the page, I renamed the "models" directory under "public".
I did not encounter errors during creation, so I will leave the renamed directories as they are now.
Is this OK?
